whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceTestIntegration.java

```diff
@@ -3236,15 +3236,15 @@ private void assertMultipleValuesRedaction(final RdapObject rdapObject, final St
 
     }
 
-    private void assertPersonalRedaction(final Entity entity, final AttributeType attribute) {
+    private void assertNotificationRedacted(final Entity entity) {
```

Comment:
Remove notify from redacted (but do test for e-mail is redacted when necessary for related entities)

Reply:
Done